### PR TITLE
Sw dal events venue linking

### PR DESCRIPTION
**Pull request by SWvheerden**

### CODE REVIEW

**Comment by mikethetike** [comment]
Required for big-neon/bn-api#32

File: src/bin.rs [hunk 1]
```diff
@@ -131,10 +131,10 @@ fn create_db_and_user(matches: &ArgMatches) {
         .expect("Password was not provided");
     println!("Creating user");
 
-    let dbConnection = DatabaseConnection::new(conn_string).unwrap();
+    let db_Connection = DatabaseConnection::new(conn_string).unwrap();
```
Comment (mikethetike) [inline src/bin.rs]: `db_connection`

Comment (mikethetike) [reply src/bin.rs]: Otherwise looks good

Comment (SWvheerden) [reply src/bin.rs]: done

**Pull request merged**